CHANGELOG.md

```diff
@@ -23,6 +23,18 @@ Our versioning strategy is as follows:
 
 * `[sitecore-jss-nextjs]` Fixed handling of ? inside square brackets [] in regex patterns to prevent incorrect escaping ([#1999](https://github.com/Sitecore/jss/pull/1999))
 
+### 🛠 Breaking Change
+
+* `[sitecore-jss]``[create-sitecore-jss]``[sitecore-jss-nextjs]``[sitecore-jss-react]``[sitecore-jss-dev-tools]``[sitecore-jss-vue]` Remove Axios ([#2006](https://github.com/Sitecore/jss/pull/2006))
```

Comment:
Let's simplify the tags, it's really a lot and especially each template is affected as well
```suggestion
* `[all packages]` `[all samples]` Remove Axios ([#2006](https://github.com/Sitecore/jss/pull/2006))
```